### DISCUSSION TITLE
@react-stately/color and @react-stately/datepicker build problem (Can't resolve '../intl/*json' @react-stately/color/src/Color.ts)

### DISCUSSION THREAD

**Discussion by Tomiczek** [adobe/react-spectrum#7243] [Q&A]
Hi folks! 
I have a strange problem with @react-state/color (and datepicker) at `3.8.0`. When I'm building my app on jenkins, I get an error
`Can't resolve '../intl/*json' @react-stately/color/src/Color.ts`.
Locally it passes fine, I have a problem only on Jenkins. When I check my local node_modules there's indeed no `intl` folder (and no json files) in @react-state/color. Maybe someone had a similar issue?

**Comment by snowystinger** [comment]
odd, it shouldn't be trying to resolve anything from 'src'

https://www.unpkg.com/browse/@react-stately/color@3.8.0/dist/Color.mjs this resolves from the `./intlStrings` file. Maybe something about source maps is misleading us. But could you put some logs into node_modules to make sure that the dist `Color` file is being loaded, any of the three (mjs, module.js, or main.js)?
